homework9/src/App.js

```diff
@@ -0,0 +1,16 @@
+import "./App.css";
+import Header from "./components/header/Header";
+import Landing from "./pages/landing/Landing";
+import Footer from "./components/footer/Footer";
```

Comment:
inside `Footer` folder create file `index.js`
and in index.js file make import like:
import { default } from './Footer'

and then your import will be as import Footer from "./components/footer" it is shorter that previosly

Reply:
also try to name your React Component from first Upper letter. It will be more clear, that it is a component

Reply:
also you can create jsconfig.json flie and add there some configuration, for example for default path and then path for your reusable component will be like components/Footer instead of ../../../../../components/Footer - it is not case from your project

Reply:
try to sort your imports.
In first group should be impors from npm
then empty line
second group - imports your reusable components, variables, functions, e.t.c (for that case it is from components folder)
empty line
then your local imports that exist only in that folder
empty line
import styles
empty line

It make your code easier for reading

empty line
then your local imports that exist only in that folder
empty line
import styles
empty line

It makes your code easier for reading

Reply:
example
import React from 'react'

import Header from "components/Header"

import LocalComponent from '../SomeComponent'

import './styles.scss'